pages/about.tsx

```diff
@@ -192,18 +193,24 @@ const AboutPage = () => {
         <p className="pb-8">
           The calldata region is the data sent to a transaction as part of a
           smart contract transaction. For example, when creating a contract,
-          calldata would be the constructor code of the new contract.
-          Transaction calldata is immutable, and can be read with instructions{' '}
+          calldata would be the constructor code of the new contract. It is
+          important to note that when a contract executes an xCALL instruction,
+          it also creates an internal transaction, with a resulting calldata
+          region in the new context. Transaction calldata is immutable, and can
```

Comment:
```suggestion
          region in the new context. Calldata is immutable, and can
```
Not sure what "transaction" calldata is, let's just call it calldata :-)

Reply:
sgtm, want me to commit?

Reply:
yes